packages/repository/test/acceptance/relations/hasMany/models/order.model.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity} from '../../../../../';
+import {model, property, belongsTo} from '../../../../../src/decorators';
+
+@model()
+export class Order extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+  })
+  id?: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  desc?: string;
+
+  @property({
+    type: 'date',
+  })
+  date?: string;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  @belongsTo({
+    target: 'Customer',
+    as: 'customer',
+    foreignKey: 'customerId',
+  })
+  customerId: string;
```

Comment:
Since "Order belongsTo Customer", there should be a repository allowing Order instances to access the owning Customer.

Usage:

```ts
const order = await order.customer.get();
await order.customer.create({/* customer data*/});
await order.customer.destroy();
```

See the full "BelongsTo" API implementation here: https://github.com/strongloop/loopback-datasource-juggler/blob/2ac5efa04a95fe6b1b5847a7dd9d6f8e743e5896/lib/relation-definition.js#L1389-L1619